lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -1897,10 +1897,11 @@ else if (lessonEntity.notPublished())
                                 boolean isAudio = mimeType.startsWith("audio/");
                                 UIComponent h5video = UIOutput.make(tableRow, (isAudio? "h5audio" : "h5video"));
                                 UIComponent h5source = UIOutput.make(tableRow, (isAudio? "h5asource" : "h5source"));
-                                if (lengthOk(height) && height.getOld().indexOf("%") < 0)
-                                h5video.decorate(new UIFreeAttributeDecorator("height", height.getOld()));
-                                if (lengthOk(width) && width.getOld().indexOf("%") < 0)
-                                h5video.decorate(new UIFreeAttributeDecorator("width", width.getOld()));
+				// HTML5 spec says % isn't legal in width, so have to use style
+                                if (lengthOk(height))
+                                h5video.decorate(new UIFreeAttributeDecorator("style", "width: " + height.getNew()));
```

Comment:
Right. I’ll add {} if I have to do that again.

I did check to see if the request not to reindent still makes sense. It was because if you reindent code it shows as a change. Unfortunately that’s true in git as well as svn. 

The result of not reindenting can be seriously unreadable code, but the reason still seems to be there.

> On Mar 29, 2016, at 3:57 PM, Sam Ottenhoff <notifications@github.com> wrote:
> 
> In lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java <https://github.com/sakaiproject/sakai/pull/2050#discussion_r57789320>:
> 
> > @@ -1897,10 +1897,11 @@ else if (lessonEntity.notPublished())
> >                                  boolean isAudio = mimeType.startsWith("audio/");
> >                                  UIComponent h5video = UIOutput.make(tableRow, (isAudio? "h5audio" : "h5video"));
> >                                  UIComponent h5source = UIOutput.make(tableRow, (isAudio? "h5asource" : "h5source"));
> > -                                if (lengthOk(height) && height.getOld().indexOf("%") < 0)
> > -                                h5video.decorate(new UIFreeAttributeDecorator("height", height.getOld()));
> > -                                if (lengthOk(width) && width.getOld().indexOf("%") < 0)
> > -                                h5video.decorate(new UIFreeAttributeDecorator("width", width.getOld()));
> > +				// HTML5 spec says % isn't legal in width, so have to use style
> > +                                if (lengthOk(height))
> > +                                h5video.decorate(new UIFreeAttributeDecorator("style", "width: " + height.getNew()));
> No indentation and no brackets on a one-line conditional is wildly difficult code reading .....
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub <https://github.com/sakaiproject/sakai/pull/2050/files/3505fe17c0cabb8a583e2d7985fbf1b51446a759#r57789320>